layouts/partials/fragments/nav.html

```diff
@@ -98,17 +98,21 @@
     </div>
   {{- end }}
 </nav>
-{{- if and .Params.breadcrumb (lt (.Params.breadcrumb_level | default 1) (sub (len ($sections | default slice)) (cond (eq $.root.Kind "home") 1 0))) }}
-  <nav aria-label="breadcrumb" class="overlay navbar navbar-expand-lg
-    {{- printf " bg-%s" $bg -}}
+{{- $breadcrumb := .Params.breadcrumb -}}
+{{- $display_breadcrumb := and $breadcrumb $breadcrumb.display -}}
+{{- $level := $breadcrumb.level -}}
+{{- $nav_bg := $breadcrumb.background | default "light" -}}
```

Comment:
Reminds me of our background non helper days ^^

Reply:
Yeah, since we removed the background helper this was necessary :D

Reply:
Did I miss something. What specifically did we remove?

Reply:
The `background` helper. We removed it, moved the snippet to `fragments` helper, added `bg` to `Scratch` and used the value inside `text-color` helper.

Reply:
Wouldn't that mean that we could get the bg from scratch instead of defaulting here again?